hpat/hiframes/hiframes_typed.py

```diff
@@ -947,30 +947,30 @@ def run_call_series_quantile_default(A):
                                                      'lt_f': series_kernels.lt_f},
                                       pre_nodes=nodes)
 
-        if func_name == 'head':
-            nodes = []
-            n_arg = self._get_arg('Series.head', rhs.args, dict(rhs.kws), 0,
-                                  'n', default=False)  # TODO: proper default handling
-            if n_arg is False:
-                n_arg = ir.Var(lhs.scope, mk_unique_var('head_n'), lhs.loc)
-                # default is 5
-                self.typemap[n_arg.name] = types.IntegerLiteral(5)
-                nodes.append(ir.Assign(
-                    ir.Const(5, lhs.loc), n_arg, lhs.loc))
-
-            data = self._get_series_data(series_var, nodes)
-            func = series_replace_funcs[func_name]
-
-            if self.typemap[series_var.name].index != types.none:
-                index = self._get_series_index(series_var, nodes)
-                func = series_replace_funcs['head_index']
-            else:
-                index = self._get_index_values(data, nodes)
-
-            name = self._get_series_name(series_var, nodes)
-
-            return self._replace_func(
-                func, (data, index, n_arg, name), pre_nodes=nodes)
+        # if func_name == 'head':
+        #     nodes = []
+        #     n_arg = self._get_arg('Series.head', rhs.args, dict(rhs.kws), 0,
+        #                           'n', default=False)  # TODO: proper default handling
+        #     if n_arg is False:
+        #         n_arg = ir.Var(lhs.scope, mk_unique_var('head_n'), lhs.loc)
+        #         # default is 5
+        #         self.typemap[n_arg.name] = types.IntegerLiteral(5)
+        #         nodes.append(ir.Assign(
+        #             ir.Const(5, lhs.loc), n_arg, lhs.loc))
+        #
+        #     data = self._get_series_data(series_var, nodes)
+        #     func = series_replace_funcs[func_name]
+        #
+        #     if self.typemap[series_var.name].index != types.none:
+        #         index = self._get_series_index(series_var, nodes)
+        #         func = series_replace_funcs['head_index']
+        #     else:
+        #         index = self._get_index_values(data, nodes)
+        #
+        #     name = self._get_series_name(series_var, nodes)
+        #
+        #     return self._replace_func(
+        #         func, (data, index, n_arg, name), pre_nodes=nodes)
```

Comment:
Dataframe.head() is based on it. Does it work with new style?

Reply:
We have not tests for Dataframe.head() (only one skiped) 
Should I write it and test?

Reply:
test_df_head1 is skipped due to 'dtype' fail. I guess it has been fixed yet, let's check. Nevertheless, test was skipped on Windows only